features/cardinal_commerce.feature

```diff
@@ -0,0 +1,117 @@
+    When User fills payment form with credit card number "<card_number>", expiration date "01/22" and cvv "123"
+    And THREEDQUERY mock response is set to "NOT_ENROLLED_U"
+    And User clicks Pay button - AUTH response is set to "<action_code>"
+    Then User will see payment status information: "<payment_status_message>"
+    And User will see that notification frame has "<color>" color
+    And AUTH and THREEDQUERY requests were sent only once with correct data
+
+    Examples:
+      | card_number      | action_code     | payment_status_message                  | color |
+      | 4111110000000401 | OK              | Payment has been successfully processed | green |
+      | 5100000000000412 | UNAUTHENTICATED | Unauthenticated                         | red   |
+      #|5100000000000412	 | DECLINE         | Decline                | red   |
+
+  @base_config @extended_tests_part_1
+  Scenario: Cardinal Commerce - check THREEDQUERY response for code: "INVALID_ACQUIRER"
+    When User fills payment form with credit card number "4111110000000211", expiration date "01/22" and cvv "123"
+    And THREEDQUERY mock response is set to "INVALID_ACQUIRER"
+    And User clicks Pay button
+    Then User will see payment status information: "Invalid acquirer for 3-D Secure"
+    And User will see that notification frame has "red" color
+    And THREEDQUERY request was sent only once with correct data
+
+  @base_config @extended_tests_part_1
+  Scenario: Cardinal Commerce - check ACS response for code: FAILURE
+    When User fills payment form with credit card number "4111110000000211", expiration date "01/22" and cvv "123"
+    And THREEDQUERY mock response is set to "ENROLLED_Y"
+    And ACS mock response is set to "FAILURE"
+    And User clicks Pay button
+    Then User will see payment status information: "Wystąpił błąd"
+    And User will see that notification frame has "red" color
+    And THREEDQUERY request was sent only once with correct data
+    And User will see that Submit button is "enabled" after payment
+
+  @base_config @cardinal_commerce
+  Scenario Outline: Cardinal Commerce - check ACS response for code: <action_code>
+    When User fills payment form with credit card number "4111110000000211", expiration date "01/22" and cvv "123"
+    And THREEDQUERY mock response is set to "ENROLLED_Y"
+    And ACS mock response is set to "<action_code>"
+    And User clicks Pay button
+    Then User will see payment status information: "<payment_status_message>"
+    And User will see that notification frame has "<color>" color
+    And AUTH and THREEDQUERY requests were sent only once with correct data
+
+    Examples:
+      | action_code | payment_status_message                  | color |
+      | NOACTION    | Payment has been successfully processed | green |
+
+  @config_submit_on_error_true @smoke_test @extended_tests_part_1
+  Scenario: Cardinal Commerce - error payment with enabled 'submit on error' process
```

Comment:
This scenario and rest with submitOnSuccess / OnError can be move to separete feature